pkgs/development/python-modules/MeshLabXML/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  lib,
+}:
+
+buildPythonPackage rec {
+  pname = "MeshLabXML";
+  version = "2018.3";
+  name = "${pname}-${version}";
```

Comment:
?
```suggestion
```

Reply:
Sorry, leftover from when I wrote this first (before `pname`).